Pod/Tests/TestStash.swift

```diff
@@ -0,0 +1,43 @@
+import UIKit
+import XCTest
+
+class TestStash: XCTestCase {
+
+  let service = "stash"
+  let password = "banana45?!"
+  let account = "John Hyperseed"
+
+  func testAddingPasswordForService() {
+    let success = Stash.setPassword(password, service: service, account: account)
+
+    XCTAssertTrue(success)
+
+    Stash.delete(service, account: account)
+  }
+
+  func testUpdatePasswordForService() {
+    let newPassword = "apple641"
```

Comment:
I think you could use `tearDown` to clean up, like delete password you've set in the each test. Then the same password  `let password = "banana45?!"` could be used everywhere.